pages/gallery/500hPa_Vorticity_Advection.ipynb

```diff
@@ -63,7 +63,7 @@
    "outputs": [],
    "source": [
     "# Open the example netCDF data\n",
-    "ncss = NCSS('https://www.ncei.noaa.gov/thredds/ncss/grid/namanl/'\n",
+    "ncss = NCSS('https://www.ncdc.noaa.gov/thredds/ncss//namanl/'\n",
```

Comment:
Let's eliminate the hard-coding of the grid vs. not (which is a thredds server issue) by actually using the catalog. So let's update to something like:

```python
cat = TDSCatalog('https://www.ncdc.noaa.gov/thredds/catalog/namanl/201604/20160416/catalog.xml')
ncss = cat.datasets['namanl_218_20160416_1800_000.grb'].subset()
```